app/src/routes/check/+server.js

```diff
@@ -103,4 +102,22 @@ function simplify_tokens(sentences) {
 			missing_arguments: missing_arguments.map(rule => rule.role_tag),
 		}
 	}
+}
+
+/** @type {Map<MessageType, number>} */
+const MESSAGE_SEVERITY = new Map([
```

Comment:
I wonder about putting `severity` on the `Message` and letting the error constructor handle these assignments...

Reply:
this will be incorporated into next commits.